README.md

```diff
@@ -41,96 +72,186 @@ pod "KlaviyoSwift"
+### Arguments
 
-```swift
-let profile = Profile(email: "junior@blob.com", firstName: "Blob", lastName: "Jr")
-KlaviyoSDK().set(profile: profile)
+The `create` method takes an event object as an argument. The event can be constructed with the following arguments:
+- [required] `name`: The name of the event you want to track, as a `EventName` enum. A list of common Klaviyo defined event metrics can be found in `Event.EventName`. You can also create custom events by using the `CustomEvent` enum case of `Event.EventName`
+- `properties`: A dictionary of properties that are specific to the event. This argument is optional.
+- `value`: A numeric value (`Double`) to associate with this event. For example, the dollar amount of a purchase.
 
-// or setting individual properties
-KlaviyoSDK().set(profileAttribute: .firstName, value: "Blob")
-KlaviyoSDK().set(profileAttribute: .lastName, value: "Jr")
-```
+## Push Notifications
+
+### Prerequisites
 
-Note that the only argument `set(profile:)` takes is a dictionary representing a customer's attributes. This is different from `trackEvent`, which can take multiple arguments.
+* An apple developer [account](https://developer.apple.com/).
+* Configure [iOS push notifications](https://help.klaviyo.com/hc/en-us/articles/360023213971) in Klaviyo account settings.
 
-## Anonymous Tracking Notice
+### Setup
 
-By default, Klaviyo will begin tracking unidentified users in your app once the SDK is initialized. This means you will be able to track events from users in your app without any user information provided. When an email or other primary identifier is provided, Klaviyo will merge the data from the anonymous user to a new identified user.
+* Enable push notification capabilities in your Xcode project. The section "Enable the push notification capability" in this [Apple developer guide](https://developer.apple.com/documentation/usernotifications/registering_your_app_with_apns#2980170) provides detailed instructions.
+* If you intend to use rich push notifications add a [Notification service extension](https://developer.apple.com/documentation/usernotifications/unnotificationserviceextension) to your xcode project.
+A notification service app extension ships as a separate bundle inside your iOS app. To add this extension to your app:
+  * Select File > New > Target in Xcode.
+  * Select the Notification Service Extension target from the iOS > Application extension section.
+  * Click Next.
+  * Specify a name and other configuration details for your app extension.
+  * Click Finish.
+
+> ⚠️ By default the deployment target of your notification service extension might be the latest iOS version and not
```

Comment:
Maybe Put a note about adjust the deployment target to match the app.

Reply:
Done.